spring-boot-persistence/src/main/java/com/baeldung/naming/MetadataExtractorIntegrator.java

```diff
@@ -0,0 +1,36 @@
+package com.baeldung.naming;
+
+import org.hibernate.boot.Metadata;
+import org.hibernate.boot.model.relational.Database;
+import org.hibernate.engine.spi.SessionFactoryImplementor;
+import org.hibernate.service.spi.SessionFactoryServiceRegistry;
+
+public class MetadataExtractorIntegrator implements org.hibernate.integrator.spi.Integrator {
```

Comment:
Its goal is to extract the name of the table associated with the entity. I didn't mention in the tutorial because for simplicity I show only the output of the statement but I didn't show the tests. Do you prefer to mention it?